improver_tests/wxcode/wxcode/test_ModalCode.py

```diff
@@ -97,12 +103,37 @@ def wxcode_series_fixture(
                     scalar_coords=time_coords,
                 )
             )
+
+        # Add a blendtime coordinate as UK weather symbols are constructed
+        # from model blended data.
+        blend_time = wxcubes[-1].coord("forecast_reference_time").copy()
+        blend_time.rename("blend_time")
+        wxcubes[-1].add_aux_coord(blend_time)
+
         if model_id_attr:
-            [c.attributes.update({MODEL_ID_ATTR: "uk_ens"}) for c in wxcubes]
-            wxcubes[0].attributes.update({MODEL_ID_ATTR: "uk_det uk_ens"})
-    return model_id_attr, wxcubes
+            if i == 0:
+                wxcubes[-1].attributes.update({MODEL_ID_ATTR: "uk_det uk_ens"})
+            else:
+                wxcubes[-1].attributes.update({MODEL_ID_ATTR: "uk_ens"})
+
+        if record_run_attr:
+            ukv_time = wxfrt - timedelta(hours=1)
+            enukx_time = wxfrt - timedelta(hours=3)
+            if i == 0:
+                wxcubes[-1].attributes.update(
+                    {
+                        RECORD_RUN_ATTR: f"uk_det:{ukv_time:{TIME_FORMAT}}:\nuk_ens:{enukx_time:{TIME_FORMAT}}:"  # noqa: E501
+                    }
+                )
+            else:
+                wxcubes[-1].attributes.update(
+                    {RECORD_RUN_ATTR: f"uk_ens:{enukx_time:{TIME_FORMAT}}:"}
+                )
+
+    return model_id_attr, record_run_attr, wxcubes
 
 
+@pytest.mark.parametrize("record_run_attr", [False])
```

Comment:
Why is this parametrized? There's only one entry.

Reply:
It is passed through to the fixture as an argument. I wasn't really sure how else to structure this as it's the parameter names that match expected arguments which pytest magically passes through to the fixture. Is there another way?

Reply:
No, probably not. I tried setting a default value on the fixture, but it complained.  I'm happy to leave this as it is.